src/woo_publications/publications/tests/test_publications_admin.py

```diff
@@ -147,9 +147,32 @@ def test_publications_admin_create(self):
                 _("This field is required."),
             )
 
+        with self.subTest(
```

Comment:
I wanted to validate the `VallidatorError` + error_message by using:
```python
form["publisher"].select(text=deactivated_organisation.naam)
``` 
But the problem with that approach is that the error message has every valid option with in the string. this makes it very annoying to test it with webtest. So instead of recreating this string myself or testing by selecting the deactivated organisation and just validating on `VallidatorError` I decided to test if the expected objects are in the queryset of the field. 

This is a little more clear that the deactivated organisation isn't an option that can be selected, but the ones that are active are.

Reply:
The `limit_choices_to` should also prevent you from being able to select the value, no? But this approach is okay - it's not as pure as it can be but it does protect us against mistakes.

Reply:
Yes it indeed does, but I thought that having tests for this would be good to have.